src/condor_scripts/condor_watch_q

```diff
@@ -1011,6 +1011,19 @@ def strip_empty_columns(rows_by_key):
     return headers, rows_by_key
 
 
+class JobId():
+    def __init__(self, cluster: int, proc: int) -> None:
+        self.cluster = cluster
+        self.proc = proc
+
+    def __str__(self) -> str:
+        return f"{self.cluster}.{self.proc}"
+
+    def __lt__(self, other) -> bool:
+        return (self.cluster < other.cluster or
+               (self.cluster == other.cluster and self.proc < other.proc))
```

Comment:
Another way to do this using tuples (your way is fine if you want to keep it):
```suggestion
    def __lt__(self, other) -> bool:
        return (self.cluster, self.proc) < (other.cluster, other.proc)
```
```python
>>> (1, 0) < (1, 0)
False
>>> (1, 0) < (1, 1)
True
>>> (1, 0) < (2, 0)
True
>>> (1, 1) < (2, 0)
True
>>> (2, 0) < (1, 3)
False
```

Reply:
Much cleaner... updated